src/MongoDB.EntityFrameworkCore/Storage/MongoDatabaseWrapper.cs

```diff
@@ -217,4 +148,148 @@ private static string GetId(IUpdateEntry entry)
             _ => throw new NotSupportedException($"Unknown type for _id property on entity type ${entry.EntityType.DisplayName()}")
         };
     }
+
+    private IEnumerable<MongoUpdate> ConvertUpdateEntriesToMongoUpdates(IEnumerable<IUpdateEntry> entries)
+    {
+        return
+            GetAllChangedRootEntries(entries)
+            .Select(ConvertUpdateEntryToMongoUpdate)
+            .Where(x => x != null)
+            .Cast<MongoUpdate>();
+    }
+
+    private MongoUpdate? ConvertUpdateEntryToMongoUpdate(IUpdateEntry entry)
+    {
+        var collectionName = entry.EntityType.GetCollectionName();
+        var id = GetId(entry);
```

Comment:
Note: `GetId` assumes the `_id` is going to be a string.

Leaving that assumption alone for now.